packages/icons/src/library/connection.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * WordPress dependencies
+ */
+import { SVG, Path, G } from '@wordpress/primitives';
+
+const connection = (
```

Comment:
From what I can see, all the icons in the package aren't filled: [link](https://wordpress.github.io/gutenberg/?path=/story/icons-icon--library). Would it make sense to export the icon without fill, just in case someone else wants to use it, and fill it with CSS in our specific use case?

Reply:
Yes, I think that's the right approach 👍

Reply:
> Would it make sense to export the icon without fill, just in case someone else wants to use it, and fill it with CSS in our specific use case?

@SantosGuillamot When you say _fill_, do you mean the purple stroke? I've [pushed a commit ](https://github.com/WordPress/gutenberg/pull/59185/commits/d571f82ef0b6797c3a2e8ab07d13e5e261f5810e) that modifies the SVG so that it's black and white by default like the other icons in the design library, and we now add the purple color using CSS.

Reply:
Yes, when I said `fill` I meant they don't have the `fill` attribute to set the color.